lib/kotkowo/application.ex

```diff
@@ -5,17 +5,24 @@ defmodule Kotkowo.Application do
   use Application
 
   alias Kotkowo.AdviceHandler
+  alias Kotkowo.Plausible.ViewPuller
 
   @impl true
   def start(_type, _args) do
-    children = [
-      # Start the Telemetry supervisor
-      KotkowoWeb.Telemetry,
-      {Phoenix.PubSub, name: Kotkowo.PubSub},
-      KotkowoWeb.Endpoint,
-      Kotkowo.PromEx,
-      Supervisor.child_spec({AdviceHandler, nil}, id: AdviceHandler, restart: :permanent)
-    ]
+    children =
+      Enum.filter(
+        [
+          KotkowoWeb.Telemetry,
+          {Phoenix.PubSub, name: Kotkowo.PubSub},
+          KotkowoWeb.Endpoint,
+          Kotkowo.PromEx,
+          Supervisor.child_spec({AdviceHandler, nil}, id: AdviceHandler, restart: :permanent),
+          maybe_view_puller()
+        ],
+        & &1
+      )
```

Comment:
Make this return maybe_view_pull() an empty list by default.
Then instead:
```suggestion
        [
          KotkowoWeb.Telemetry,
          {Phoenix.PubSub, name: Kotkowo.PubSub},
          KotkowoWeb.Endpoint,
          Kotkowo.PromEx,
          Supervisor.child_spec({AdviceHandler, nil}, id: AdviceHandler, restart: :permanent)
        ] ++ maybe_view_puller()
```